modules/core/src/test/java/org/apache/ignite/internal/processors/performancestatistics/AbstractPerformanceStatisticsTest.java

```diff
@@ -42,6 +47,10 @@
     /** */
     public static final long TIMEOUT = 30_000;
 
+    /** File performance statistics pattern {@link FilePerformanceStatisticsReader}. */
+    static final Pattern FILE_PATTERN = Pattern.compile(
```

Comment:
Why do we need to copy-paste regexp here?

Reply:
fixed